src/appleseed.shaders/src/appleseed/as_standard_surface.osl

```diff
@@ -35,10 +35,7 @@ shader as_standard_surface
     string as_maya_classification = "drawdb/shader/surface:rendernode/appleseed/surface:shader/surface:swatch/AppleseedRenderSwatch",
     string help = "Standard Surface Shader",
     string icon = "asStandardSurface",
-    int as_maya_type_id = 0x001279d4,
-    string as_max_class_id = "651059681 943276521",
```

Comment:
Do we want to remove these lines?

Reply:
No we don't, i'm not sure how that ended up there.